pkg/config/config.go

```diff
@@ -229,11 +225,14 @@ func GetJSON(data interface{}) (string, error) {
 
 // OpenPiperFile provides functionality to retrieve configuration via file or http
 func OpenPiperFile(name string) (io.ReadCloser, error) {
-	//ToDo: support also https as source
 	if !strings.HasPrefix(name, "http") {
 		return os.Open(name)
 	}
-	return nil, fmt.Errorf("file location not yet supported for '%v'", name)
+
+	// support http(s) urls next to file path - url cannot be protected
+	client := http.Client{}
+	response, err := client.SendRequest("GET", name, nil, nil, nil)
```

Comment:
See my other PR about HTTP timeouts. The file better be completely downloaded in 10 seconds. ;-)

Reply:
We could increase to 20 but I guess 10 for a simple text file should be good enough, what do you think?

Reply:
You can leave it as it is and approve my other PR... then it would error after *stalling* for 10 seconds.